### PR TITLE
Ash/feature

### PR DESCRIPTION
**Pull request by AshlandWest**

### CODE REVIEW

**Comment by bryant-ferguson** [comment]
Can you squash all of these to one commit and then rebase onto master?
Or perhaps squash into two commits.
One for the donation page and one for the products page.

File: hifireg/registration/static/scripts/donate-page-scripts.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+        // // const charLimit = function(event) {
```
Comment (bryant-ferguson) [inline hifireg/registration/static/scripts/donate-page-scripts.js]: This entire file looks like it needs to be deleted as it's entirely commented out.

File: hifireg/registration/templates/utils/form_fields_donate.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{% load static %}
```
Comment (bryant-ferguson) [inline hifireg/registration/templates/utils/form_fields_donate.html:1]: Delete this file and use "form_fields.html" instead. You can attach the javascript handler without needing to inline the "onchange" property which I think is the only difference between this file and "form_fields.html"

Comment (AshlandWest) [reply hifireg/registration/templates/utils/form_fields_donate.html:1]: I have no idea what this file is other than what it's name tells me. I assume it is linked somewhere that would need to be updated?

Comment (AshlandWest) [reply hifireg/registration/templates/utils/form_fields_donate.html:1]: Done

**Pull request closed without merging**